### PR TITLE
Rename ProcessBundleProgressMetadataRequest to MonitoringInfosMetadataRequest. And rename ProcessBundleProgressMetadataResponse to MonitoringInfosMetadataResponse

### PR DESCRIPTION
**Pull request by ajamato**
Rename ProcessBundleProgressMetadataRequest to MonitoringInfosRequest.
And rename ProcessBundleProgressMetadataResponse to MonitoringInfosResponse
And yapf format

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/#make-reviewers-job-easier).

Post-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

Lang | SDK | Dataflow | Flink | Samza | Spark | Twister2
--- | --- | --- | --- | --- | --- | ---
Go | [Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Go/lastCompletedBuild/) | --- | [Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Go_VR_Flink/lastCompletedBuild/) | --- | [Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Go_VR_Spark/lastCompletedBuild/) | ---
Java | [Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Java/lastCompletedBuild/) | [Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow/lastCompletedBuild/)<br>[Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow_Java11/lastCompletedBuild/) | [Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink/lastCompletedBuild/)<br>[Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink_Java11/lastCompletedBuild/)<br>[Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Java_PVR_Flink_Batch/lastCompletedBuild/)<br>[Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Java_PVR_Flink_Streaming/lastCompletedBuild/) | [Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza/lastCompletedBuild/) | [Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark/lastCompletedBuild/)<br>[Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Java_PVR_Spark_Batch/lastCompletedBuild/)<br>[Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_SparkStructuredStreaming/lastCompletedBuild/) | [Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Twister2/lastCompletedBuild/)
Python | [Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Python36/lastCompletedBuild/)<br>[Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Python37/lastCompletedBuild/)<br>[Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Python38/lastCompletedBuild/) | [Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/)<br>[Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Py_VR_Dataflow_V2/lastCompletedBuild/)<br>[Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | [Build Status](https://ci-beam.apache.org/job/beam_PreCommit_Python_PVR_Flink_Cron/lastCompletedBuild/)<br>[Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Python_VR_Flink/lastCompletedBuild/) | --- | [Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Python_VR_Spark/lastCompletedBuild/) | ---
XLang | [Build Status](https://ci-beam.apache.org/job/beam_PostCommit_XVR_Direct/lastCompletedBuild/) | --- | [Build Status](https://ci-beam.apache.org/job/beam_PostCommit_XVR_Flink/lastCompletedBuild/) | --- | [Build Status](https://ci-beam.apache.org/job/beam_PostCommit_XVR_Spark/lastCompletedBuild/) | ---

Pre-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

--- |Java | Python | Go | Website | Whitespace | Typescript
--- | --- | --- | --- | --- | --- | ---
Non-portable | [Build Status](https://ci-beam.apache.org/job/beam_PreCommit_Java_Cron/lastCompletedBuild/) | [Build Status](https://ci-beam.apache.org/job/beam_PreCommit_Python_Cron/lastCompletedBuild/)<br>[Build Status](https://ci-beam.apache.org/job/beam_PreCommit_PythonLint_Cron/lastCompletedBuild/)<br>[Build Status](https://ci-beam.apache.org/job/beam_PreCommit_PythonDocker_Cron/lastCompletedBuild/) <br>[Build Status](https://ci-beam.apache.org/job/beam_PreCommit_PythonDocs_Cron/lastCompletedBuild/) | [Build Status](https://ci-beam.apache.org/job/beam_PreCommit_Go_Cron/lastCompletedBuild/) | [Build Status](https://ci-beam.apache.org/job/beam_PreCommit_Website_Cron/lastCompletedBuild/) | [Build Status](https://ci-beam.apache.org/job/beam_PreCommit_Whitespace_Cron/lastCompletedBuild/) | [Build Status](https://ci-beam.apache.org/job/beam_PreCommit_Typescript_Cron/lastCompletedBuild/)
Portable | --- | [Build Status](https://ci-beam.apache.org/job/beam_PreCommit_Portable_Python_Cron/lastCompletedBuild/) | --- | --- | --- | ---

See [.test-infra/jenkins/README](https://github.com/apache/beam/blob/master/.test-infra/jenkins/README.md) for trigger phrase, status and link of all Jenkins jobs.

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by ajamato** [comment]
R: @lukecwik

File: model/fn-execution/src/main/proto/beam_fn_api.proto [hunk 1]
```diff
@@ -312,12 +312,12 @@ message ProcessBundleProgressRequest {
 // An SDK can report metrics using an identifier that only contains the
```
Comment (lukecwik) [inline model/fn-execution/src/main/proto/beam_fn_api.proto:312]: I can't provide a suggestion for the first line, please adjust to `A request to provide full MonitoringInfo for a given id.`

File: model/fn-execution/src/main/proto/beam_fn_api.proto [hunk 2]
```diff
@@ -312,12 +312,12 @@ message ProcessBundleProgressRequest {
...
 // associated payload. A runner who wants to receive the full metrics
 // information can request all the monitoring metadata via a
-// ProcessBundleProgressMetadataRequest providing a list of ids as necessary.
+// MonitoringInfosMetadataRequest providing a list of ids as necessary.
 //
 // The SDK is allowed to reuse the identifiers across multiple bundles as long
 // as the MonitoringInfo could be reconstructed fully by overwriting its
 // payload field with the bytes specified here.
```
Comment (lukecwik) [inline model/fn-execution/src/main/proto/beam_fn_api.proto:319]:
```suggestion
// The MonitoringInfo ids are scoped to the associated control connection. For
// example, an SDK may reuse the ids across multiple bundles.
```

File: model/fn-execution/src/main/proto/beam_fn_api.proto [hunk 3]
```diff
@@ -312,12 +312,12 @@ message ProcessBundleProgressRequest {
...
-message ProcessBundleProgressMetadataRequest {
+message MonitoringInfosMetadataRequest {
   // A list of ids for which the full MonitoringInfo is requested for.
```
Comment (lukecwik) [inline model/fn-execution/src/main/proto/beam_fn_api.proto:321]:
```suggestion
  // A list of ids for which MonitoringInfo are requested. All but the payload field will be populated.
```

File: model/fn-execution/src/main/proto/beam_fn_api.proto [hunk 4]
```diff
@@ -348,12 +348,12 @@ message ProcessBundleProgressResponse {
...
 // information can request all the monitoring metadata via a
-// ProcessBundleProgressMetadataRequest providing a list of ids as necessary.
+// MonitoringInfosMetadataRequest providing a list of ids as necessary.
 //
 // The SDK is allowed to reuse the identifiers across multiple bundles as long
 // as the MonitoringInfo could be reconstructed fully by overwriting its
 // payload field with the bytes specified here.
-message ProcessBundleProgressMetadataResponse {
+message MonitoringInfosMetadataResponse {
   // A mapping from an identifier to the full metrics information.
```
Comment (lukecwik) [inline model/fn-execution/src/main/proto/beam_fn_api.proto:357]:
```suggestion
  // A mapping from a requested identifier to a MonitoringInfo. All fields except for
  // the payload of the MonitoringInfo will be specified.
```

File: model/fn-execution/src/main/proto/beam_fn_api.proto [hunk 5]
```diff
@@ -348,12 +348,12 @@ message ProcessBundleProgressResponse {
 // An SDK can report metrics using an identifier that only contains the
 // associated payload. A runner who wants to receive the full metrics
 // information can request all the monitoring metadata via a
-// ProcessBundleProgressMetadataRequest providing a list of ids as necessary.
+// MonitoringInfosMetadataRequest providing a list of ids as necessary.
 //
 // The SDK is allowed to reuse the identifiers across multiple bundles as long
 // as the MonitoringInfo could be reconstructed fully by overwriting its
 // payload field with the bytes specified here.
```
Comment (lukecwik) [inline model/fn-execution/src/main/proto/beam_fn_api.proto:355]:
```suggestion
// The MonitoringInfo ids are scoped to the associated control connection. For
// examplem an SDK may reuse the ids across multiple bundles.
```

**Review by lukecwik — APPROVED** [review]
Please adjust comments based upon suggestions and then can merge.

Please associate a JIRA with this task (or overarching goal)

**Comment by ajamato** [comment]
R: @lostluck Please review. I removed the go changes. Please follow up and include those with the generated go protos in a separate PR as discussed

**Comment by lukecwik** [comment]
Run Python PreCommit

**Comment by lostluck** [comment]
Ack. It looks like the head process for Go Protos has change enough that the instructions need to be adjusted and updated as well. I don't anticipate any additional changes to the SDK as a result other than incorporating these renames, but the Go proto files themselves will be updated drastically to handle the change to the v2 version of Go protocol buffers.

**Comment by lostluck** [comment]
I've now got a compiling and passing version of the go protos. It's in PR https://github.com/apache/beam/pull/13115 which will be updated once this PR is in.

**Comment by lukecwik** [comment]
Run Python PreCommit

**Comment by lukecwik** [comment]
Run Python PreCommit

**Comment by lukecwik** [comment]
Run Python PreCommit

**Comment by ajamato** [comment]
Thanks for the review, looks like its ready now

**Comment by lostluck** [comment]
I'm ready to update  #13115 with this once it's merged in.

**Comment by lukecwik** [comment]
@lostluck Was waiting on tests to pass and they finally did so you should be able to update #13115 now.

**Comment by lostluck** [comment]
Thanks! date #13115  should be updated now.

**Pull request merged**